lxd/container.go

```diff
@@ -243,6 +243,25 @@ func containerValidDeviceConfigKey(t, k string) bool {
 	}
 }
 
+func validateRawIdmap(rawIdmap string) error {
```

Comment:
Should probably be renamed to something more descriptive like `allowedUnprivilegedOnlyMap()` to avoid us accidentally using it somewhere else in the codebase or "fixing" it by allowing mapping uid 0 later on.

What's the reason for also preventing mapping to another uid/gid inside the container? I can't think of a security reason to prevent that.

Reply:
No particular reason.  We only need support for identity mapping so I figured I would relax the restrictions just enough to allow only that.

Reply:
Ok cool.

Can you update the branch to rename the function and drop that check then?

Then we should be good to send that through Jenkins and merge.